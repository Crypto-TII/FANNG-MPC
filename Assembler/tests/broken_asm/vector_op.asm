vldms 0, s20, 8192 # 40
vldms r42, s20, 8192 # 40
vldms
vldms 42
vldms 45, 13
vldms 45, s20, r14
vldms 45, s32, 8192,
vldms 45,,
