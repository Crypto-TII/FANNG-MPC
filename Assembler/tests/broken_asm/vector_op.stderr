error: expected an operand`
 --> tests/broken_asm/vector_op.asm:7:21
  |
7 | vldms 45, s32, 8192,
  |                      in argument list here
  |
error: expected an operand`
 --> tests/broken_asm/vector_op.asm:8:10
  |
8 | vldms 45,,
  |           in argument list here
  |
error: expected an operand`
 --> tests/broken_asm/vector_op.asm:8:11
  |
8 | vldms 45,,
  |            in argument list here
  |
error: invalid vector length: expected value between 2 and 2^16, got `0`
 --> tests/broken_asm/vector_op.asm:1:7
  |
1 | vldms 0, s20, 8192 # 40
  |       ^ for vector length here
  |
error: expected integer, got `r42`
 --> tests/broken_asm/vector_op.asm:2:7
  |
2 | vldms r42, s20, 8192 # 40
  |       ^^^ for operand here
  |
error: expected at least 1 arguments
 --> tests/broken_asm/vector_op.asm:3:1
  |
3 | vldms
  | ^^^^^
  |
error: invalid number of arguments
 --> tests/broken_asm/vector_op.asm:4:1
  |
4 | vldms 42
  | ^^^^^^^^ The number of arguments given did not match what the assembly instruction requires
  |
error: invalid number of arguments
 --> tests/broken_asm/vector_op.asm:5:1
  |
5 | vldms 45, 13
  | ^^^^^^^^^^^^ The number of arguments given did not match what the assembly instruction requires
  |
error: invalid number of arguments
 --> tests/broken_asm/vector_op.asm:7:1
  |
7 | vldms 45, s32, 8192,
  | ^^^^^^^^^^^^^^^^^^^^ The number of arguments given did not match what the assembly instruction requires
  |